spec/system/admin/alert_type_management_spec.rb

```diff
@@ -51,6 +50,7 @@
       visit root_path
       click_on 'Manage'
       click_on 'Alert Types'
+      allow_any_instance_of(AlertType).to receive(:available_charts).and_return([['chart description A', :chart_a], ['chart description B', :chart_b]])
```

Comment:
Always a bit reluctant to add these, but happy to consider other alternatives...

Reply:
The alert type factory should create all alerts with a dummy/test alert class that conforms to all the interfaces we want to use in the tests.

Reply:
Possibly easier to think about when we have "System" alerts that we own in our code.